examples/smart-contract-test-bench/src/lib.rs

```diff
@@ -0,0 +1,831 @@
+//! Smart contract test bench
```

Comment:
Maybe provide a bit more context here. That is, the reason for this contract's existence.